web/src/app/services/ServiceDetails.js

```diff
@@ -4,6 +4,7 @@ import gql from 'graphql-tag'
 import { useQuery } from 'react-apollo'
 import { Redirect } from 'react-router-dom'
 import _ from 'lodash-es'
+import Tooltip from '@material-ui/core/Tooltip'
```

Comment:
is this import still needed (I don't see it referenced)?  If so, not sure why `es-lint` didn't catch this?